CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Changelog
 
-## unreleased
+## `0.6.0` (2023-07-17)
```

Comment:
Is this date right?

Reply:
D'oh! Updated to today's date. This PR now needs a new approval.